.github/workflows/node.yml

```diff
@@ -0,0 +1,30 @@
+name: Node
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - stable*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use node ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies & build
+        run: |
+          npm ci
+          npm run build --if-present
+      - name: Check webpack build changes
```

Comment:
Can we add a big note that this is only required for shipped apps? In general people should ot commit their bundles to the repos.

Reply:
Sure thing :)
It's in progress, I'm adding phpunit ;)

Reply:
Wohooo :tada:

Reply:
I'm stuck, I cannot get server to reach the mariadb service for some reason, I really don't understand :'(

Reply:
But it's close, please see other comments